packages/react-date-time/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@patternfly/react-date-time",
+  "version": "0.0.0",
+  "description": "This library provides a set of React date and time components for use with the PatternFly 4",
+  "main": "dist/js/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/js/index.d.ts",
+  "patternfly:src": "src/",
+  "sideEffects": [
+    "*.css",
+    "*.scss"
+  ],
+  "publishConfig": {
+    "access": "public",
+    "tag": "prerelease"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/patternfly/patternfly-react.git"
+  },
+  "keywords": [
+    "react",
+    "patternfly",
+    "date",
+    "time"
+  ],
+  "author": "Red Hat",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/patternfly/patternfly-react/issues"
+  },
+  "homepage": "https://github.com/patternfly/patternfly-react/tree/master/packages/react-date-time#readme",
+  "scripts": {
+    "clean": "rimraf dist"
+  },
+  "dependencies": {
+    "@patternfly/patternfly": "4.44.1",
+    "@patternfly/react-core": "^4.55.1",
+    "@patternfly/react-icons": "^4.7.7",
+    "@patternfly/react-styles": "^4.7.5",
+    "@patternfly/react-tokens": "^4.9.9"
+  },
+  "peerDependencies": {
+    "react": "^16.8.0",
+    "react-dom": "^16.8.0"
+  },
+  "devDependencies": {
+    "rimraf": "^2.6.2",
+    "typescript": "^3.8.3",
+    "flatpickr": "^4.6.6",
+    "moment": "^2.27.0"
```

Comment:
These should be listed as dependencies. Also, please pin them to exact versions instead of using the `^` semver.

I'm hesitant to add `moment`. I'm not sure how well it code-splits, and many of its functions have been replaced by browser standards. They recently stated the [project has reached legacy status.](https://momentjs.com/docs/#/-project-status/) If all we need is ISO 8601 parsing [Date.prototype.toISOString()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString) and [Date.parse()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse) should be supported consistently in all ES5 browsers (PatternFly currently targets ES6).

Reply:
I believe we will be adding moment when we get to adding the timepicker especially because of the desires to build in timezone pickers and such. So I can probably not rely on it for the date picker, but I felt that since we will be using it for the time picker and the date and time picker will eventually be used in tandem, it made sense to make sure the date picker was thinking like moment. but you're right that it probably isn't necessary yet.

Reply:
Timezone locale formatting has come a long way since `moment` was created and is also supported in ES5 browsers: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString. 

If there's a set of functions you need implemented I'm happy to find lighter alternatives to `moment`!

Reply:
A good set of resources on how to replace moment is here: https://github.com/you-dont-need/You-Dont-Need-Momentjs/blob/master/README.md

If the native APIs really don't cut it, Luxon is a much more lightweight alternative made by the same developer of Moment: https://moment.github.io/luxon/

Reply:
Although @redallen the concern about using a heavy library here is maybe less of an issue for the general consumer since this is going into a separate `@patternfly/react-date-time` package. If for some reason we determine we really do need something like Moment at least it won't be in the bundle for those who only use core.